src/BuildIt.Forms/BuildIt.Forms.Controls/Enums.cs

```diff
@@ -25,4 +25,22 @@ public enum CameraFocusMode
         /// </summary>
         Manual
     }
+
+    /// <summary>
+    /// Defines the camera preview is displayed
+    /// </summary>
+    public enum CameraPreviewAspect
+    {
+        /// <summary>
+        /// Scale the preview to fit the view. Some parts may be left empty (letter boxing)
+        /// </summary>
+        // Note: aspect fit is the default (as opposed to none) to be consistent with how the
+        // Aspect enum is defined in Xamarin Forms to reduce cognitive load of devs
+        AspectFit,
+
+        /// <summary>
+        /// Scale the preview so it exactly fills the view. Scaling may not be in uniform in X and Y
+        /// </summary>
+        Fill
```

Comment:
We need to add AspectFill - stretches to fill entire space but maintains aspect ratio (UniformToFill in UWP speak)